frontend-new/components/LinkIconButton.tsx

```diff
@@ -0,0 +1,62 @@
+import { Box, Button, ButtonProps } from '@mui/material'
+import { FC, ReactNode, PropsWithChildren } from 'react'
+
+import { media } from 'app/styles/theme'
+
+interface Props extends PropsWithChildren {
+  href?: string
+  icon: ReactNode
+  sx?: ButtonProps['sx']
+}
+
+export const LinkIconButton: FC<Props> = ({ href = '#', icon, children, sx }) => {
+  return (
+    <Button
+      href={href}
+      target='_blank'
+      sx={{
+        backgroundColor: '#FB815B',
+        height: 48,
+        pr: 1,
+        [media.TABLET]: {
```

Comment:
reorganize to have MQs below other styles in the obj

Reply:
agree, moved it everywhere to the bottom